src/backend/web/handlers/team.py

```diff
@@ -119,3 +122,27 @@ def team_list(page: int) -> str:
         "current_page": page,
     }
     return render_template("team_list.html", template_values)
+
+
+@cached_public
+def avatar_list(year: Optional[Year] = None) -> Response:
+    year = year or SeasonHelper.get_current_season()
+
+    valid_years = list(range(2018, SeasonHelper.get_max_year() + 1))
+    valid_years.remove(2021)  # No avatars in 2021 :(
+
+    if year not in valid_years:
+        abort(404)
+
+    avatars_future = Media.query(Media.media_type_enum
+                                 == MediaType.AVATAR, Media.year == year).fetch_async()
+    avatars = sorted(avatars_future.get_result(),
+                     key=lambda a: int(a.references[0].id()[3:]))
+
+    template_values = {
+        "year": year,
+        "valid_years": valid_years,
```

Comment:
Do we use this `valid_years` in the template?

Reply:
Oops, that was for a then-unfinished year switcher thing. How do you feel about this?

https://user-images.githubusercontent.com/34525547/169712363-810984c2-6c9d-4e88-8e0a-8b806434759f.mov

(I could also stick this in a separate PR if it's out of scope)